src/bin/good-fences.rs

```diff
@@ -1,32 +1,106 @@
 extern crate good_fences_rs_core; // Optional in Rust 2018
 extern crate serde_json;
+use clap::{self, Parser};
```

Comment:
just `use clap` here and reference `Parser` as `clap::Parser` for consistency when addressing symbols

Reply:
`Parser` needs to be imported in order to execute `Cli::Parse()`